### PR TITLE
Fix Search Input 

### PR DESCRIPTION
**Pull request by ntenczar**
image
image

### CODE REVIEW

File: public/assets/icons/common.svg [hunk 1]
```diff
@@ -9,6 +9,7 @@
...
   <symbol id="close" viewBox="53 53 758 758"><path d="M806 211c3.333-4.666 5-9 5-13s-1.667-8.334-5-13L679 58c-4.667-3.333-9-5-13-5s-8.333 1.667-13 5L432 279 211 58c-4.667-3.333-9-5-13-5s-8.333 1.667-13 5L58 185c-3.333 4.666-5 9-5 13s1.667 8.334 5 13l221 221L58 653c-3.333 4.667-5 9-5 13s1.667 8.333 5 13l127 127c4.667 3.333 9 5 13 5s8.333-1.667 13-5l221-221 221 221c4.667 3.333 9 5 13 5s8.333-1.667 13-5l127-127c3.333-4.667 5-9 5-13s-1.667-8.333-5-13L585 432l221-221z"/></symbol>
   <symbol id="chat" viewBox="0 0 23 32"><path d="M23.273 12.364q0 3.933-3.407 6.694t-8.23 2.76q-.942 0-1.965-.134-3.043 3.044-6.275 3.044-.323 0-.458-.296-.108-.323.108-.54.162-.16.417-.484t.727-1.32.605-2.074q-2.236-1.32-3.515-3.327T0 12.364Q0 8.43 3.41 5.67t8.23-2.76 8.228 2.76 3.407 6.694z"/></symbol>
+  <symbol id="checkmark" viewBox="0 0 23 32"><path d="M23.138 8.108l-14.061 14.061q-0.189 0.135-0.35 0.135t-0.35-0.135l-8.242-8.242q-0.35-0.35 0-0.7l3.394-3.394q0.35-0.35 0.7 0l4.498 4.525 10.317-10.343q0.35-0.35 0.7 0l3.394 3.394q0.35 0.35 0 0.7z"></path></symbol>
```
Comment (meagonqz) [inline public/assets/icons/common.svg]: y dis

**Comment by meagonqz** [comment]
Are there any more of these `ss-icons`?

File: public/assets/icons/common.svg (hunk 1 above)
Comment (ntenczar) [inline public/assets/icons/common.svg]: need it for a different component

Comment (ntenczar) [inline public/assets/icons/common.svg]: image

**Comment by ntenczar** [comment]
I think we may have culled the last that we need imminently

**Pull request merged**